README.md

```diff
@@ -8,109 +8,104 @@ You can start with forking this repository and off you go.
 
-## Setup
+It's helpful for us if you include some notes to explain the decisions you make.
 
-We recommend using NVM for managing node versions on your machine.
+# Instructions
 
-```
-brew install nvm
-```
+We'd like you to use React. We have provided a skeleton for the React application and included [SASS](http://sass-lang.com/) for styling. We have also provided some useful constants in [constants.scss](./src/constants.scss). Feel free to add any additional libraries which you think are needed for the task.
 
-...and installing the required node version with the following command:
+We have also included a mock API which is documented below.
 
-```
-nvm install
-```
+We'd like you to build two pages. We've provided designs for mobile so don't worry about making it look good on desktop.
 
-This repo is using Yarn as it's package manager.
-
-```
-brew install yarn
-```
-
-Installing packages:
-
-```
-yarn install
-```
-
-## Client implementation
-
-We would like you to use React. Feel free to add whatever libraries you think are needed for the task's completion
-We have provided you with a barebones react project to work on.
-To run the client:
-
-```
-yarn start
-```
-
-We have added [SASS](http://sass-lang.com/) to the client which allows you to re-use variables in stylesheets. We have also provided some useful constants in [constants.scss](./src/constants.scss).
+# New appointment page
 
 ### Design
+![New Appointment Design](./design/new-appointment.png)
 
-The final design of the client should look and feel like the design files we have provided in `/design`.
-
-### Pages
+### Data
 
-* list appointments: `design/appointments.png`
-* book an appointment: `design/new-appointment.png`
+You will need to use 3 API's for this page.
 
-## API
-
-### `GET /users/:userId`
+### `GET /users/1`
 
 Returns a `user` object with the following info:
 
 * `firstName`: `String` of the user's first name.
 * `lastName`: `String` of the user's last name.
 * `avatar`: `String` Base64 representation of the user's avatar image.
 * `dateOfBirth`: `Date` The user's Date of Birth.
-  _NOTE_: `userId` can be any Integer number; the results will not be affected but it **must** be present in the request's endpoint.
+  _NOTE_: `userId` can be any Integer number
 
 ### `GET /availableSlots`
 
 Returns:
 
 * `Array` of `Date ISO 8601` that represent the available appointment slots.
 
-### `GET /appointments/?userId=:userId`
+### `POST /appointments`
 
-Returns an `Array` of `appointment` objects that represent all booked appointments for the given userId
+Expects an object in the request's body with the following information:
 
-* `id`: `Int` The unique id of this appointment.
 * `userId`: `Int` The user's ID that this appointment is for.
 * `dateTime`: The user's selected time slot as a `Date ISO 8601`.
 * `notes`: `String` Any notes the user added when booking. _note_: not required, can be null.
+  The endpoint responds with a 200 status on success.
```

Comment:
why is this line between two bullets in the list of response fields?

Reply:
Not sure, copy+pasted. Will fix!